app/test_main.py

```diff
@@ -1,3 +1,53 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+def test_get_human_age_zero() -> None:
```

Comment:
Consider using @pytest.mark.parametrize to avoid repeating similar test cases with different inputs. This will make your tests more concise and maintainable.